reference/routes-yaml.md

```diff
@@ -178,6 +177,19 @@ You will see a route such as `https://*---add-theme-vmwklxcpbi6zq.eu.platform.sh
 
 [You can find detailed information about caching here](cache.html).
 
+##WebSocket routes
+
+To use WebSocket on a route, `cache` must be disabled. WebSocket is incompatible with buffering which is required for cache.
+Here is an example to define a route for serving WebSocket:
+
+```yaml
+"http://{default}/ws":
+    type: upstream
+    upstream: "websocket:http"
```

Comment:
Where does this upstream come from?  Is websocket the name of a a project, or a forwarding type?  (This is a general problem of the documentation that it makes the foo:foo syntax very unclear, but in this particular case I have no idea what these magic values are or where they come from.)